pkg/apis/core/validation/validation.go

```diff
@@ -1965,15 +2002,51 @@ func ValidatePersistentVolumeStatusUpdate(newPv, oldPv *core.PersistentVolume) f
 	return allErrs
 }
 
+// PersistentVolumeClaimValidationOptions contains the different settings for PersistentVolumeClaim validation
+type PersistentVolumeClaimValidationOptions struct {
```

Comment:
nit: have a single PersistentVolumeClaimSpecValidationOptions, since you are using it for PVC and PVCTemplate.

We can split into different structs in the future if we see the need, but likely referring to the spec is enough.

Reply:
Done. Also renamed PV validation options to `PersistentVolumeSpecValidationOptions` to be consistent (using "Spec").